atoma-p2p/src/service.rs

```diff
@@ -244,42 +250,38 @@ impl AtomaP2pNode {
                 );
                 AtomaP2pNodeError::GossipsubSubscriptionError(e)
             })?;
-        swarm.listen_on(config.listen_addr.parse()?).map_err(|e| {
-            error!(
-                target = "atoma-p2p",
-                event = "listen_on_error",
-                listen_addr = config.listen_addr,
-                error = %e,
-                "Failed to listen on address"
-            );
-            AtomaP2pNodeError::SwarmListenOnError(e)
-        })?;
+        swarm
+            .listen_on(config.listen_addr.parse().unwrap())
```

Comment:
Maybe use `expect` here ? Also might be good to handle the error instead

Reply:
Good call, I've updated in cc5d4d5 If that works feel free to merge